src/main/resources/specification/specification.yaml

```diff
@@ -103,35 +107,77 @@ components:
       properties:
         id:
           type: string
-          description:
-            The id given by the harvest system
+          description: The id given by the harvest system
         publisher:
-          $ref: "#/components/schemas/Publisher"
+          $ref: '#/components/schemas/Publisher'
         harvestSourceUri:
           type: string
-          description:
-            The source where the record was harvested from
+          description: The source where the record was harvested from
         harvest:
-          $ref: "#/components/schemas/Harvest"
+          $ref: '#/components/schemas/Harvest'
         title:
-          type: string
-          description:
-            The title of the information model
+          $ref: '#/components/schemas/TextLanguage'
         schema:
           type: string
-          description:
-            The model itself, expressed in JSON-SCHEMA
+          description: 'The model itself, expressed in JSON-SCHEMA'
+        rootObject:
+          $ref: '#/components/schemas/ObjectType'
+        objectType:
+          type: array
+          items:
+            $ref: '#/components/schemas/ObjectType'
+        codeType:
+          type: array
+          items:
+            $ref: '#/components/schemas/CodeType'
+        dataType:
+          type: array
+          items:
+            $ref: '#/components/schemas/DataType'
+        simpleType:
+          type: array
+          items:
+            $ref: '#/components/schemas/SimpleType'
```

Comment:
Er det mulig å legge til en "s" bokstav på slutten av disse for å indikere at de er arrays ved å bare lese feltnavn

Reply:
Ja, det er gjort